airflow/upgrade/rules/no_additional_args_in_operators.py

```diff
@@ -43,7 +42,11 @@ def check(self, dags_folder=None):
         old_level = logger.level
         try:
             logger.setLevel(logging.ERROR)
-            dagbag = DagBag(dag_folder=os.devnull, include_examples=False, store_serialized_dags=False)
+            dagbag = DagBag(
+                dag_folder=conf.get("core", "dags_folder"),
```

Comment:
```suggestion
                dag_folder=dags_folder,
```

Since we already get the config on L39